src/components/Image/Image.tsx

```diff
@@ -31,9 +31,9 @@ export const imageSizesArray = [
   "large",
 ] as const;
 export const imageTypesArray = ["default", "circle"] as const;
-export type ImageRatios = (typeof imageRatiosArray)[number];
-export type ImageSizes = (typeof imageSizesArray)[number];
-export type ImageTypes = (typeof imageTypesArray)[number];
```

Comment:
Automatic eslint update?

Reply:
I am not sure where the parentheses initially came from.  I feel like they appeared when I merged with `development` at one point.

Reply:
I think they appeared from Diego's update. We briefly chatted about that.